pkg/module/go_get_fetcher.go

```diff
@@ -55,23 +54,10 @@ func (g *goGetFetcher) Fetch(ctx context.Context, mod, ver string) (*storage.Ver
 	ctx, span := observ.StartSpan(ctx, op.String())
 	defer span.End()
 
-	// setup the GOPATH
-	goPathRoot, err := afero.TempDir(g.fs, "", "athens")
+	m, cleanFunc, err := downloadModule(g.goTool, g.fs, mod, ver)
```

Comment:
the function `Fetch` implements the contract of `module.Fetcher`, and i think it's okay to flat out the implementation inside of it instead of abstract it, so I feel that we should keep it the way it is.